md_api/src/main/java/com/wipro/ats/bdre/md/api/HaltJob.java

```diff
@@ -89,7 +87,8 @@ public HaltJobInfo execute(String[] params) {
             //The HaltJob completes even if sending message fails
             try {
                 BasicConfigurator.configure();
-                new StatusNotification(haltMessage, MDConfig.getProperty("status-notification.halt-queue"));
+                StatusNotification statusNotification = new StatusNotification(haltMessage, MDConfig.getProperty("status-notification.halt-queue"));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this instantiation of "StatusNotification". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2440)